libs/ui/src/lib/button/button.component.html

```diff
@@ -1,3 +1,28 @@
-<button type="button" class="cvi-button__button" [attr.disabled]="disabled ? disabled : null" (click)="onClick()" #buttonRef>
-  <ng-content></ng-content>
+<button
+  type="button"
+  class="{{ iconName && 'cvi-button__icon-button' }} cvi-button__button"
```

Comment:
According to BEM rules, two classes with different `__...` subwords in them can't coexist (this means two BEM elements are attached to the same HTML node, but only one can be at a time). The `cvi-button__icon-button` class is not used in the CSS codebase, but if it would be, to fix this we could turn it into a BEM modifier for an already added BEM element: `cvi-button__button--has-icon`.